federatedrec/neural_cf/hetero_ncf/hetero_ncf_arbiter.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#
+#  Copyright 2019 The FATE Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from arch.api.utils import log_utils
+from federatedml.util import consts
+from federatedml.optim.convergence import converge_func_factory
+from fate_flow.entity.metric import MetricType, MetricMeta, Metric
+from federatedrec.neural_cf.hetero_ncf.hetero_ncf_base import HeteroNCFBase
+
+LOGGER = log_utils.getLogger()
+
+
+class HeteroNCFArbiter(HeteroNCFBase):
+    def __init__(self):
+        super(HeteroNCFArbiter, self).__init__()
+        self.role = consts.ARBITER
+
+    def _init_model(self, params):
+        super()._init_model(params)
+        early_stop = self.model_param.early_stop
+        self.converge_func = converge_func_factory(early_stop.converge_func, early_stop.eps).is_converge
+        self.loss_consumed = early_stop.converge_func != "weight_diff"
+
+    def callback_loss(self, iter_num, loss):
+        """
+        call back function of loss and metrics.
+        :param iter_num: iter number.
+        :param loss: loss type.
+        :return:
+        """
+        metric_meta = MetricMeta(name='train',
+                                 metric_type=MetricType.LOSS,
```

Comment:
Hetero MetricType.LOSS is not right, may be "LOSS", or it will not show loss curve on fate-board